pkg/local_object_storage/blobstor/fstree/fstree.go

```diff
@@ -346,6 +379,38 @@ func (t *FSTree) GetBytes(addr oid.Address) ([]byte, error) {
 	return dec, nil
 }
 
+func extractCombinedObject(addr oid.ID, data []byte) ([]byte, error) {
```

Comment:
`addr` is not `oid.ID`, would not name it this way

Reply:
This naming is already used in some places and OID is an address in some ways.

Reply:
Fixed though.